File: Command/Golang/Image.go
package main

import (
	"fmt"
	"math"
)

type Image struct {
	ImagePosition int
	ImageSize	  float64
}

func (image *Image) rotate(degree int) {
	image.ImagePosition = int(math.Abs(float64(image.ImagePosition + degree))) % 360
}

func (image *Image) resize(percentage int) {
	image.ImageSize *= float64(percentage) / 100.0
}

func (image *Image) inspect() {
	fmt.Println(
		"Image Position:", 
		image.ImagePosition, 
		"\n",
		"Image Size:", 
		image.ImageSize,
		"\n",
		"\n",
	)
}

File: Command/Dart/Image.dart
class Image {
  int? _imagePosition;
  double? _imageSize;

  Image(this._imagePosition, this._imageSize);

  void rotate(int degree) {
    _imagePosition = (_imagePosition! + degree).abs() % 360;
  }

  void resize(int percentage) {
    _imageSize = _imageSize! * ((percentage / 100) as double?)!;
  }

  void inspect() {
    print("""
      Image Position: $_imagePosition
      Image Size    : $_imageSize
    """);
  }
}
